test/e2e/network/networking.go

```diff
@@ -41,7 +41,7 @@ import (
 // host. An error will be returned if the host is not reachable from the pod.
 //
 // An empty nodeName will use the schedule to choose where the pod is executed.
-func checkConnectivityToHost(f *framework.Framework, nodeName, podName, host string, port, timeout int) error {
+func CheckConnectivityToHost(f *framework.Framework, nodeName, podName, host string, port, timeout int) error {
```

Comment:
it seems is better to not export this function and keep using each one per platform